charts/apisix/templates/_helpers.tpl

```diff
@@ -96,17 +96,3 @@ prometheus:
 {{- define "apisix.pluginAttrs" -}}
 {{- merge .Values.pluginAttrs (include "apisix.basePluginAttrs" . | fromYaml) | toYaml -}}
 {{- end -}}
-
-{{- define "apisix.podAntiAffinity" -}}
```

Comment:
Why delete this one?

Reply:
not need for this template function. as kube-scheduler will take care of this.

Reply:
yes